app/services/physical_holdings_markup_builder.rb

```diff
@@ -293,21 +293,31 @@ def self.location_services_block(adapter, holding_id, location_rules, link, hold
 
   # Generate a request label based upon the holding location
   # @param location_rules [Hash] the location for the holding
+  # @param adapter [HoldingRequestsAdapter] adapter for the SolrDocument and Bibdata API
   # @return [String] the label
-  def self.request_label(location_rules)
+  def self.request_label(location_rules, adapter)
```

Comment:
Curious about the decision to pass adapter here instead of the just columbia id.   Do you see the adapter being use more in the future?

Reply:
I was being lazy and didn't want to update or write more tests for the calling method. Do you think I should change it?

Reply:
It's ok, it just makes it look like we are dependent on more than just the id.  I would not worry about it for the moment though...